### DISCUSSION TITLE
sunshine.conf - keybindings

### DISCUSSION THREAD

**Discussion by bytebool** [LizardByte/community#189] [Sunshine] [answered]
Has something changed recently with regards to keybindings in the sunshine.conf file?
I`ve previous used the following keybindings with success, but for some reason, recently the keybindings are not taking effect:

UK qwerty
```
keybindings = [
   0xde, 0xc0,
   0xc0, 0xdf,
   0xdc, 0xde,
   0xe2, 0xdc
]
```
I`ve double checked all the values and they still seem right.
Tried to even reinstall, but still no success.

Any help anyone can lend on the matter would be very much appreciated.

**Comment by ld892012** [comment, marked as answer]
The keyboard layout issue is fixed from v0.17.0 i guess (accordingly to [this issue](https://github.com/LizardByte/Sunshine/issues/493))
I have removed keybindings from my settings since this version and my keyboard layout works (FR azerty)
Maybe you can try to remove your keybindings settings and try your keys as is ^^

**Reply by bytebool** [reply]
Well, damn. Why didnt I think to try that first?
Thanks buddy, that seems to have fixed the issue.

**Reply by FlorianWolters** [reply]
I'm using Sunshine v0.21.0 with a German QWERTZ keyboard on a NVIDIA Shield TV Pro (2019) and the keyboard is QWERTY when streaming via Moonlight. Is this a regression and should I create a new issue? Also refer to my comment https://github.com/LizardByte/Sunshine/issues/1392#issuecomment-1925362069.
